src/Congo.WebApi/Controllers/ProductsController.cs

```diff
@@ -14,16 +14,20 @@ namespace Congo.WebApi.Controllers
     public class ProductsController : Controller
     {
         private readonly IMediator _mediator;
+        private readonly IMapper _mapper;
 
-        public ProductsController(IMediator mediator)
+
+        public ProductsController(IMediator mediator, IMapper mapper)
         {
             _mediator = mediator;
+            _mapper = mapper;
         }
 
         [HttpGet]
-        public async Task<IEnumerable<Product>> Get()
+        public async Task<ActionResult<ProductResponse>> Get()
```

Comment:
```suggestion
        public async Task<ActionResult<ProductResponse[]>> Get()
```

Incorrect return type

Reply:
I would prefer IEnumerable
```suggestion
        public async Task<ActionResult<IEnumerable<ProductResponse>>> Get()
```

Reply:
If we go with that, the map statement should also be updated to map to `IEnumerable<ProductResponse>` of course :)